kakao-shop/src/components/molecules/CartList.jsx

```diff
@@ -127,27 +157,23 @@ const CartList = ({ data }) => {
             {Array.isArray(cartItems) &&
               cartItems.map((item) => {
                 if (item.carts.length === 0) return null;
-                return <CartItem key={item.id} item={item} onChange={handleOnChangeCount} onDelete={handleOnDelete} />;
+                return (
+                  <CartItem
+                    key={item.id}
+                    item={item}
+                    onChange={handleOnChangeCount}
+                    onDelete={handleOnDelete}
+                  />
+                );
               })}
           </div>
           <div className="flex justify-between font-bold text-lg bg-white p-4 mb-1 border border-solid border-gray-200">
             <span>주문 예상금액</span>
-            <span className=" text-kakao-blue text-xl">{`${comma(totalPrice)}원`} </span>
+            <span className=" text-kakao-blue text-xl">
+              {`${comma(totalPrice)}원`}{" "}
```

Comment:
```suggestion
              {`${comma(totalPrice)}원`}
```
문자열로 공간을 주는 것보다 스타일로 적용해보는 것도 고려해보세요